### PR TITLE
[#2] Build home page layout

### PR DESCRIPTION
**Pull request by jjannetty**
@kirkland and @nmertaugh there is little more I can do until we have decided on content to put in the site. I will make that an action Item of mine to come up with some marketing copy, also anything specific you want me to highlight or say, let me know.

In the meantime if one of you could take a look at what I have done, it would be much appreciated.

### CODE REVIEW

File: app/assets/javascripts/home.coffee [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  target = $('.js-parallax')
+
+  $(window).scroll ->
+    scrollDistance = $(window).scrollTop() - target.offset().top
+    parallaxDistance = scrollDistance/2
+
+    target.css 'top': parallaxDistance
+
+$(document).ready ready
+
```
Comment (kirkland) [inline app/assets/javascripts/home.coffee]: Too many blank lines at end of file here!

File: app/assets/stylesheets/common/base.css.scss [hunk 2]
```diff
@@ -0,0 +1,57 @@
+body, html {
+  color: $gray-dark;
```
Comment (kirkland) [inline app/assets/stylesheets/common/base.css.scss]: Alphabetize? Or make your own rules and crown yourself king of stylesheets!

**Comment by kirkland** [comment]
In a different PR, @nmertaugh was in favor of not using translations, and I conceded because it didn't seem to matter either way. What's your opinion on the matter?

**Comment by jjannetty** [comment]
I am fine with saying fuck translations

File: app/assets/javascripts/home.coffee (hunk 1 above)
Comment (jjannetty) [inline app/assets/javascripts/home.coffee]: removed

**Comment by jjannetty** [comment]
did that stuff you said above... @nmertaugh you want to pass an admiring eye over my PR, or do you want it to be ship ship sharoo

**Comment by nmertaugh** [comment]
:shipit: :rocket:

**Pull request merged**